docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -305,6 +305,111 @@ document
 
 </PlatformSection>
 
+### Loading Strategies
+
+Because the `feedbackIntegration` is a user-facing integration, we offer two loading strategies that have implications on bundle size.
+
+<Note>
+In every case, we recommend using `feedbackIntegration` in your `Sentry.init` call.
+</Note>
+
+All of the code examples on this page use `feedbackIntegration` as a default because it is available whether you have chosen the CDN or NPM installation method. However, the implementation of `feedbackIntegration` is different depending on the installation method. For NPM users `feedbackIntegration` is an alias of `feedbackSyncIntegration`. For CDN users, `feedbackIntegration` is an alias of `feedbackAsyncIntegration`.
+
+#### `feedbackSyncIntegration`
+
+If you have installed the SDK with NPM, this loading strategy is used by default. This strategy is not available if you have installed the SDK via the CDN.
+
+This integration includes all the code needed to render the "Send Feedback" button, as well as the form that is triggered when the button is clicked. Choosing this loading strategy means accepting the largest bundle size in your application. The benefit of this is that the feedback widget is guaranteed to work when the user interacts with it.
```

Comment:
> The benefit of this is that the feedback widget is guaranteed to work when the user interacts with it.

We might want to rephrase this, it won't "work" in terms of sending the feedback if the user has adblock.